src/Options/Definitions.js

```diff
@@ -168,6 +168,11 @@ module.exports.ParseServerOptions = {
     help: 'Adapter module for the files sub-system',
     action: parsers.moduleOrObjectParser,
   },
+  fileUpload: {
```

Comment:
Did you manually add these lines to the `Definitions.js` file? This file is recreated by the nodejs `definitions` script, based on the definition that you add in `/src/Options/index.js`. This will also create these options for the CLI. Note that since we have sub-parameters here, you'll need to
- create an interface in `/src/Options/index.js` where you add these sub-parameters and
- add an entry to `getENVPrefix` in `buildConfigDefinitions.js`

Reply:
Oh, thanks for the pointers. I didn't realise that.

I will need to do that on the Security Vulnerability PR. I'll do both that tomorrow. Thanks!

Reply:
You can take a look at an existing config object, for example search for the term `PARSE_SERVER_EXPERIMENTAL_IDEMPOTENCY` and you'll find hints for how it's done.

Reply:
Thanks mate. I didn't realise the blaring 

"This code has been generated by resources/buildConfigDefinitions.js
Do not edit manually, but update Options/index.js"

My bad. How does that look now? I'll also have to change the MFA code as well.

Reply:
I've also added `encryptionKey` to Options/index.js, as when I ran `definitions`, it got removed.

Reply:
We shouldn't mix these things. I would see that in a separate PR.